assets/dev/js/editor/utils/jquery-html5-dnd.js

```diff
@@ -205,14 +205,20 @@
 				return;
 			}
 
-			// Fix placeholder placement for Container with `flex-direction: row`.
 			const $currentElement = $( currentElement ),
 				isRowContainer = $currentElement.parents( '.e-container--row' ).length,
-				isFirstInsert = $currentElement.hasClass( 'elementor-first-add' );
+				isFirstInsert = $currentElement.hasClass( 'elementor-first-add' ),
+				isInnerContainer = $currentElement.hasClass( 'e-container__inner' ),
+				$rowTargetElement = isInnerContainer ? $currentElement.closest( '.e-container' ) : $currentElement,
```

Comment:
This line should be in line 220 - you only need it if it is a row container, so declare the variable in that `if` block.

Reply:
updated